charts/cluster-scanner/templates/_helpers.tpl

```diff
@@ -118,7 +118,7 @@ ise_js_producer_subject: "analysis.sboms"
 Generates config for Redis, if available
 */}}
 {{- define "cluster-scanner.redisCacheConfig" }}
-{{- if eq .Values.imageSbomExtractor.cache.type "redis" }}
+{{- if and (contains "distributed" .Values.imageSbomExtractor.cache.type) (.Values.imageSbomExtractor.cache.redis) }}
```

Comment:
I am fine with it but as an improvement, I guess we should fail if the `cache.type="distributed"` and `cache.redis` is not provided.